schema.graphql

```diff
@@ -37,11 +37,17 @@ type Address implements Node {
     """Returns the elements in the list that come before the specified cursor"""
     before: String
 
+    """If supplied, filters by the specific event"""
+    event: String
```

Comment:
are we sure we can query this from the blockchain?

Reply:
yes it is the way it's been designed, as the idea is that the event will be created in a way that it can be viewed in any dApp